libcontainer/specconv/spec_linux.go

```diff
@@ -467,6 +516,28 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 				MemBwSchema:   spec.Linux.IntelRdt.MemBwSchema,
 			}
 		}
+		if spec.Linux.MemoryPolicy != nil &&
+			(spec.Linux.MemoryPolicy.Mode != "" ||
+				spec.Linux.MemoryPolicy.Nodes != "" ||
+				len(spec.Linux.MemoryPolicy.Flags) > 0) {
+			var ok bool
+			specMp := spec.Linux.MemoryPolicy
+			confMp := &configs.LinuxMemoryPolicy{}
+			if confMp.Mode, ok = mpolModeMap[specMp.Mode]; !ok {
```

Comment:
I really don't like this `if a, ok = ...` form as it can easily be misread as creating a scoped variable. If you want to assign to the variable just put it on a separate line.

Reply:
Fixed to make variable scopes clear. This also fixes leaking err to non-local variable.